### ISSUE TITLE
Let users specify which simulator to use

### ISSUE THREAD

**Issue by gfontenot** [gfontenot/vim-xcodebuild#8]
ideally, users would be able to (at least) pick an OS version to run against. This seems fairly important. We could (should) probably go further, and let users decide which simulator to run against.

**Comment by gfontenot** [comment]
I mean "device" when I say "simulator".

**Comment by gfontenot** [comment]
I actually think I did mean simulator, at least to start with. We're not actually specifying OS versions anywhere right now. I'm going to rename this to match that expectation.

Basically what I think we'd want:
- Provide a sane default for the simulator. We can naively default to like `iPhone 6s` right now, but that won't be enough once we have finished #40 and #41.
- Allow users to store a local default in case we're not picking the right one (like we do with projects/schemes/workspaces)
- Pass the selected simulator name to the `run_ios_app` script to be consistent (ENV variable?)
- Allow users to override the main simulator on the fly with a local command (`Xsimulator`)
- Allow users to pass a simulator name to `Xrun` to run in a different simulator

I'm going to implement this the simplest way possible for right now, which is to use Good Ol' Freeform Text. I'll open up a couple more issues to track improvements.
